libs/libarchfpga/src/arch_util.cpp

```diff
@@ -1034,7 +1089,7 @@ void CreateModelLibrary(t_arch* arch) {
     model_library[1].next = &model_library[2];
     model_library[1].outputs = nullptr;
 
-    //LATCH
+    //LATCH triggered at RISING EDGE
```

Comment:
Make enumerated constants to get rid of these magic numbers.

Reply:
Added `InternalModel` enum